client/my-sites/hosting/phpmyadmin-card/restore-db-password.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { getHttpData, httpData, requestHttpData } from 'state/data-layer/http-data';
+import { http } from 'state/data-layer/wpcom-http/actions';
+import Dialog from 'components/dialog';
+import { connect } from 'react-redux';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { successNotice } from 'state/notices/actions';
+
+const requestId = siteId => `restore-db-password-${ siteId }`;
+
+export const restoreDatabasePassword = siteId =>
```

Comment:
Should we move these to `state/data-getters`

Reply:
I assumed that path is only for getters, so I refrained from move it there, given this is not a getter. Although there are more things than just getters right now in that folder. We could maybe rename the folder to something else.